cam2ground_extrinsic:
  rotation: [1, 0, 0, 
            0, 1, 0, 
            0, 0, 1]
  translation: [0, 0, 0]

lidar2cam_extrinsic:
  rotation: [0, 1, 0, 
             0, 0, -1, 
             -1, 0, 0]
  translation: [0.00, 0.0508, 0.0502]

lidarData2lidarSensor_extrinsic:
  rotation: [-1, 0, 0,
            0, -1, 0,
            0, 0, 1]
  translation: [0, 0, 0.03618]

